README.rst

```diff
@@ -194,6 +194,21 @@ is enabled or not.
 It is disabled by default because `OpenWISP <http://openwisp.org>`_ does not use
 this feature of `django-organizations <https://github.com/bennylope/django-organizations>`_ yet.
 
+Multitenant mixins
+------------
+
+These are mixins which make a ModelAdmin class multitenant: users will see only the objects related to the
+organizations they are associated with.
+
+* **MultitenantAdminMixin**: adding this mixin to a ``ModelAdmin`` class will make it multitenant.
+  Set ``multitenant_shared_relations`` to the list of parameters you wish to have only organization
+  specific options.
```

Comment:
incredible how unclear this line is and I don't even remember what that does exactly :-(, I have opened an issue to improve this in the future: https://github.com/openwisp/openwisp-users/issues/58